### DISCUSSION TITLE
assertSnapshot() failing even though the record is the same as the screenshot

### DISCUSSION THREAD

**Discussion by bwidulski-mdsol** [pointfreeco/swift-snapshot-testing#611] [Q&A]
I'm currently running tests that fail on certain screenshots even though they're exactly the same. I went to an image difference website and it didn't detect any differences between the record screenshot and the current one, but still fails. Anyone know why this may be?

**Comment by alfonsogarza** [comment]
Have you tried comparing the images with another program or util to confirm they are indeed `exactly the same`?

Either way it's almost impossible to help without more details. Are the images from the same platform / device? \
#424 is an example where images 'look' identical but are not when captured from simulators in different machines.
